src/sparseml/core/model/pytorch.py

```diff
@@ -41,9 +41,12 @@ class ModifiableModelPyTorch(ModifiableModel[Module, Module, Parameter]):
     """
 
     def __init__(
-        self, framework: Optional[Framework] = None, model: Optional[Module] = None
+        self,
+        framework: Optional[Framework] = None,
+        model: Optional[Module] = None,
+        layer_prefix: str = "",
```

Comment:
This should default to None if we don't provide a prefix, otherwise I believe it'll fail down the line in `cache_attention_inputs`

Reply:
Good catch; updated!